### PR TITLE
Process all active repos

### PR DESCRIPTION
**Pull request by akinnane**

### CODE REVIEW

File: audit_lambda.py [hunk 1]
```diff
@@ -189,7 +189,7 @@ def analyse_pull_request_status(today):
     now = arrow.utcnow()
     pull_requests = storage.read_json(f"{today}/data/activity_prs.json")
     repositories = storage.read_json(f"{today}/data/repositories.json")
-    for repo in repositories.public:
+    for repo in repositories.active:
```
Comment (alice-carr) [inline audit_lambda.py]: For style, we should maybe keep consistent with bracket vs dot notation

Comment (akinnane) [reply audit_lambda.py]: Agreed!

Comment (akinnane) [reply audit_lambda.py]: I've made those changes.

**Comment by 0atman** [comment]
LGTM, not much needed :ok_hand:

**Pull request merged**